pdfminer/layout.py

```diff
@@ -72,8 +74,21 @@ def __init__(self,
         self.boxes_flow = boxes_flow
         self.detect_vertical = detect_vertical
         self.all_texts = all_texts
+
+        self._validate()
         return
 
+    def _validate(self):
+        # Validate boxes_flow
```

Comment:
Use docstring with `"""`

Reply:
Hm, this was actually intended as a "this part of the code is validating the boxes_flow parameter", since perhaps in the future we will validate more things.

I personally think it's silly to add a docstring to a function called `_validate` which would basically be just `"""validate the params"""`, so if it works for you I think I'll just remove this comment?